build.gradle.kts

```diff
@@ -18,13 +18,13 @@ subprojects {
             if (project.findProperty("composeCompilerReports") == "true") {
                 freeCompilerArgs += listOf(
                     "-P",
-                    "plugin:androidx.compose.compiler.plugins.kotlin:reportsDestination=${project.buildDir.absolutePath}/compose_compiler"
+                    "plugin:androidx.compose.compiler.plugins.kotlin:reportsDestination=${project.layout.buildDirectory.asFile.get()}/compose_compiler"
```

Comment:
Out of curiosity, why you changed this line? Could you please mention your reference?

Reply:
![img](https://github.com/Kaaveh/ComposeNews/assets/1156370/f539a947-2def-435a-a128-31c85d4291c0)

Reply:
Interesting! In my Android studio I haven't this warning (Hedghog Beta 6). What's your Android studio Version?

Reply:
It's deprecated by Gradle